webpack.config.js

```diff
@@ -1,43 +1,43 @@
-
-const path = require("path");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
 
 module.exports = {
-  entry: "./src/index.js",
+  entry: './src/index.js',
   output: {
-    path: path.join(__dirname, "/dist"),
-    filename: "index_bundle.js"
+    path: path.join(__dirname, '/dist'),
+    filename: 'index_bundle.js',
+    publicPath: '/'
+  },
+  devServer: {
```

Comment:
Hi @gbols. So as not to get a major merge conflict when @JohnMadakin work is merged, remove this line. It has already been catered for by him.